app/src/main/java/edu/uc/hornunjb/weatherwear/dto/Weather.kt

```diff
@@ -40,48 +16,12 @@ class Weather {
     var icon: String? = null
 }
 
-class Clouds {
-    @SerializedName("all")
-    var all: Float = 0.toFloat()
-}
```

Comment:
I thought it would be better practice to put this class in its own file. What I found was it is better for classes to be alone.

Reply:
If used in other classes, or even the potential to be used in other classes, move it outside.  The vast majority of the time, there's a chance it could be used in another class, so it is good practice to move it outside.

That also helps with extension.  It's hard to extend an inner class.